lib/widgets/HomeScreen/home_screen.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:classroom_app/widgets/CustomButton/custom_button.dart';
+import 'package:flutter/material.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: Padding(
+        padding: const EdgeInsets.all(28.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Center(
+              child: Image.asset(
+                'assets/images/overviewPage_image.png',
+              ),
+            ),
+            const SizedBox(
```

Comment:
Replace the hardcoded `SizedBox` heights with `Spacer` or `Expanded widgets` to dynamically distribute space

Reply:
to get the desired space between two or more widgets it should use sizebox.